md_api/src/main/java/com/wipro/ats/bdre/md/api/HaltJob.java

```diff
@@ -42,6 +43,17 @@
             {"batchmarking", "batch-marking", "Batch Marking of the batches enqueued for downstream"}
     };
 
+
+    public HaltJob() {
+        AutowireCapableBeanFactory acbFactory = getAutowireCapableBeanFactory();
+        acbFactory.autowireBean(this);
+    }
+    @Autowired
+    private JobDAO jobDAO;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1213)